spec/cb/clients/cover_letters_spec.rb

```diff
@@ -20,60 +22,37 @@ module Cb
           'created_data' => '2016-03-02T14:29:30.617998-05:00', 'modified_date' => '2016-03-02T14:29:30.617998-05:00'
       }
     end
-    let(:response) do
-      {
-          'data' => [ data ].flatten,
-          'page' => 1,
-          'page_size' => 1,
-          'total' => 1
-      }
-    end
-    let(:error_response) do
-      {
-          'errors' => [ data ].flatten,
-          'page' => -1,
-          'page_size' => 0,
-          'total' => 0
-      }
-    end
-    let(:headers) do
-      {
-          'Accept'=>'application/json',
-          'Accept-Encoding'=>'deflate, gzip',
-          'Authorization'=>'Bearer token',
-          'Content-Type' => 'application/json',
-          'Developerkey'=> Cb.configuration.dev_key
-      }
-    end
+    let(:data){ [cover_letter] }
 
     context '#create' do
-      let(:data){ cover_letter }
+      let(:uri) {"https://api.careerbuilder.com/consumer/coverletters?developerkey=#{ Cb.configuration.dev_key }&outputjson=true" }
+
+      let(:data){ [cover_letter] }
```

Comment:
Do you need to redefine `data` here since you're not redefining `cover_letter` in this context? `data` is already defined on line 25.